src/api/java/com/minecolonies/api/util/Tuple.java

```diff
@@ -1,15 +1,31 @@
 package com.minecolonies.api.util;
 
+import org.jetbrains.annotations.Nullable;
+
 /**
  * Our own tuple implementation with hashcode and equals.
  * @param <A> The first obj.
  * @param <B> The second obj.
  */
-public class Tuple<A, B> extends net.minecraft.util.Tuple<A, B>
+public class Tuple<A, B>
```

Comment:
This class is not needed.
Use an instance of TCustomHash<Map/Set> where you need this support, and just use the original class.

Reply:
I also want elements of the tuple to be nullable, the original class doesn't support this either

Reply:
(We use this tuple at a few other places too, but used to extend just the minecraft tuple, in this PR I removed this "extend" to make it nullable).

Reply:
Okey then